.solhint.json

```diff
@@ -0,0 +1,54 @@
+{
+    "extends": "default",
+    "rules": {
+      "reentrancy": "error",
+      "avoid-sha3": "error",
+      "avoid-suicide": "error",
+      "avoid-throw": "error",
+      "func-visibility": "error",
+      "state-visibility": "error",
+      "check-send-result": "error",
+      "avoid-call-value": "error",
+      "compiler-fixed": "error",
+      "compiler-gt-0_4": "error",
+      "no-complex-fallback": "error",
+      "mark-callable-contracts": "error",
+      "multiple-sends": "error",
+      "no-simple-event-func-name": "error",
+      "avoid-tx-origin": "error",
+      "no-inline-assembly": "error",
+      "not-rely-on-block-hash": "error",
+      "avoid-low-level-calls": "error",
+
+      "func-name-mixedcase": "error",
+      "func-param-name-mixedcase": "error",
+      "var-name-mixedcase": "error",
+      "event-name-camelcase": "error",
+      "const-name-snakecase": "error",
+      "modifier-name-mixedcase": "error",
+      "contract-name-camelcase": "error",
+      "use-forbidden-name": "error",
+      "visibility-modifier-order": "error",
+      "imports-on-top": "error",
+      "two-lines-top-level-separator": "error",
+      "func-order": "error",
+      "quotes": ["error", "double"],
+      "no-mix-tabs-and-spaces": "error",
+      "indent": ["error", 4],
+      "bracket-align": "error",
+      "array-declaration-spaces": "error",
+      "separate-by-one-line-in-contract": "error",
+      "expression-indent": "error",
+      "statement-indent": "error",
+      "space-after-comma": "error",
+      "no-spaces-before-semicolon": "error",
+
+      "max-line-length": ["error", 120],
+      "payable-fallback": "error",
+      "no-empty-blocks": "error",
+      "no-unused-vars": "error",
+      "function-max-lines": ["error", 45],
+      "code-complexity": ["error", 7],
+      "max-states-count": ["error", 15]
+    }
+}
```

Comment:
See below: 👇 

It's not letting me write this comment on the file itself - but can we keep the TRANSLATIONS_CONTRIBUTION_GUIDELINES.md name? My thinking here with explicit naming was to make it clear for non technical contributors and to be distinct from code contributions which may (or may not) end up being a thing later.

Reply:
Then we need both 🤷‍♀️

Reply:
(Context: I had thought that the title was from when it was part of another repo.)

Reply:
Yeah, the title is explicit because it's meant to be consumed by non tech contributors. Sure, I'll add a contrib guidelines md 👍

Reply:
I may have added them 😛 Feel free to expand on those (see update comment in the main thread under `Conversation`)